python-sdk/src/astro/databases/aws/redshift.py

```diff
@@ -7,13 +7,29 @@
 from sqlalchemy import create_engine
 from sqlalchemy.engine.base import Engine
 
-from astro.constants import DEFAULT_CHUNK_SIZE, LoadExistStrategy, MergeConflictStrategy
+from astro.constants import (
+    DEFAULT_CHUNK_SIZE,
+    FileLocation,
+    FileType,
+    LoadExistStrategy,
+    MergeConflictStrategy,
+)
 from astro.databases.base import BaseDatabase
+from astro.exceptions import DatabaseCustomError
 from astro.files import File
 from astro.settings import REDSHIFT_SCHEMA
 from astro.sql.table import Metadata, Table
 
 DEFAULT_CONN_ID = RedshiftSQLHook.default_conn_name
+NATIVE_PATHS_SUPPORTED_FILE_TYPES = {
+    FileType.CSV: "CSV",
+    # By default, COPY attempts to match all columns in the target table to JSON field name keys.
+    # With this option, matching is case-sensitive. Column names in Amazon Redshift tables are always lowercase,
+    # so when you use the 'auto' option, matching JSON field names must also be lowercase.
+    # Refer: https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-format.html#copy-json
+    FileType.JSON: "JSON 'auto'",
```

Comment:
@pankajkoti can you add a test case where we have mixed casing in cols names for load_file operator? use - tests/data/homes_upper.csv. 

I just want to make sure this works with mixed col casing.

Reply:
Added a test. Also using` json 'auto ignorecase' ` instead of `json 'auto'` option to allow non-lower case columns as mentioned here https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-format.html#copy-json